py3status/modules/gpmdp.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Display currently playing song from Google Play Music Desktop Player.
+
+Configuration parameters:
+    - cache_timeout : how often we refresh this module in seconds (5s default)
+
+Requires
+    - gpmdp
+    - gpmdp-remote
+
+@author Aaron Fields https://twitter.com/spirotot
+@license BSD
+"""
+
+# import your useful libs here
+from time import time
+from subprocess import check_output
+import shlex
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+
+    cache_timeout = 5
+    format = u'♫ {info}'
+
+    @staticmethod
+    def run_cmd(cmd):
+        return check_output(shlex.split('gpmdp-remote %s' % (cmd))).decode('utf-8').strip()
+
+    def gpmdp(self, i3s_output_list, i3s_config):
+        if self.run_cmd('status') == 'Paused':
+            result = ''
+        else:
+            cmds = ['info', 'title', 'artist', 'album', 'status', 'current',
+                    'status', 'time_total', 'time_current', 'album_art']
```

Comment:
Oops, fixed.

On Wed, Jul 6, 2016 at 8:27 AM tobes <notifications@github.com> wrote:

> In py3status/modules/gpmdp.py
> <https://github.com/ultrabug/py3status/pull/333#discussion_r69720495>:
>
> > +    """
> > +    # available configuration parameters
> > +
> > +    cache_timeout = 5
> > +    format = u'♫ {info}'
> > +
> > +    @staticmethod
> > +    def run_cmd(cmd):
> > +        return check_output(shlex.split('gpmdp-remote %s' % (cmd))).decode('utf-8').strip()
> > +
> > +    def gpmdp(self, i3s_output_list, i3s_config):
> > +        if self.run_cmd('status') == 'Paused':
> > +            result = ''
> > +        else:
> > +            cmds = ['info', 'title', 'artist', 'album', 'status', 'current',
> > +                    'status', 'time_total', 'time_current', 'album_art']
>
> you have 'status' twice
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ultrabug/py3status/pull/333/files/387ce7ecec4248febb5a10454909dd6eb250afc8#r69720495>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAtBebHDCvR0ISOgvI3--H-26Fap5dJdks5qS58_gaJpZM4I17CO>
> .
>
-- 

-Aaron